cluster/gke/util.sh

```diff
@@ -305,42 +327,51 @@ function detect-node-names {
 
   NODE_NAMES=()
   for group in "${NODE_INSTANCE_GROUPS[@]:-}"; do
+    # We can't simply use --zone "${ZONE}" as ZONE may not be set (e.g. when REGION is set).
+    local igm_zone=$(gcloud compute instance-groups managed list "${group}" --format='value(zone)')
```

Comment:
does that format flag works like that?
Recently there were problems in multiple places that we had to pass csv[] or sth like that in the format.